scaffolder-templates/gitlab-workflow/template.yaml

```diff
@@ -0,0 +1,450 @@
+                        - quayOrgName
+                        - quayRepoName
+              required:
+                - namespace
+                - argocdNamespace
+                - persistencePSQLSecretName
+                - persistencePSQLUserKey
+                - persistencePSQLPasswordKey
+                - persistencePSQLServiceName
+                - persistencePSQLServiceNamespace
+                - persistencePSQLServicePort
+                - persistencePSQLDatabaseName
+  steps:
+    - id: createQuayRepository
+      name: Create Quay Repository
+      action: quay:create-repository
+      if: ${{ parameters.quayConfig == 'Create New Repository' }}
+      input:
+        baseUrl: ${{ parameters.baseUrl }}
+        token: ${{ parameters.token }}
+        name: ${{ parameters.quayRepoName }}
+        visibility: ${{ parameters.visibility }}
+        description: ${{ parameters.newRepoDescription }}
+        repoKind: ${{ parameters.repoKind }}
+        namespace: ${{ parameters.quayNamespace }}
+
+    - id: workflowCodeTemplate
+      name: Generating the Workflow Source Code and Catalog Info Component
+      action: fetch:template
+      input:
+        # "Relative path is not allowed to refer to a directory outside its parent"
+        url: ../basic-workflow/skeleton
+        values:
+          namespace: ${{ parameters.namespace }}
+          argocdNamespace: ${{ parameters.argocdNamespace }}
+          orgName: ${{ parameters.orgName }}
+          repoName: ${{ parameters.repoName }}
+          owner: ${{ parameters.owner }}
+          system: ${{ parameters.system }}
+          description: ${{ parameters.description }}
+          groupId: io.janus.workflow
+          artifactId: ${{ parameters.workflowId }}
+          version: 1.0.0-SNAPSHOT
+          workflowId: ${{ parameters.workflowId }}
+          workflowType: ${{ parameters.workflowType }}
+          infrastructureWorkflowId: ${{ parameters.infrastructureWorkflowId }}
+          sourceControl: gitlab.com
+          applicationType: workflow-project
+          lifecycle: development
+          quayOrgName: ${{ parameters.quayOrgName }}
+          quayRepoName: ${{ parameters.quayRepoName }}
+        targetPath: workflow
+    - id: renameFiles
+      action: fs:rename
+      name: Rename files
+      input:
+        files:
+          - from: workflow/src/main/resources/${{ parameters.workflowType }}-template.sw.yaml
+            to: workflow/src/main/resources/${{ parameters.workflowId }}.sw.yaml
+            overwrite: false
+    - id: deleteFiles
+      action: fs:delete
+      name: Delete files
+      input:
+        files:
+          - workflow/src/main/resources/assessment-template.sw.yaml
+          - workflow/src/main/resources/infrastructure-template.sw.yaml
+    - id: buildCodeTemplate
+      name: Generating the Build Code for the Workflow
+      action: fetch:template
+      if: ${{ parameters.CI == 'tekton_argocd' }}
+      input:
+        url: ../build
```

Comment:
this will copy the whole content of the build folder, including the `.github` folder that is useless for this, could you add a step that delete unneeded directories?
I guess some would goes for github target and the `.gitlab-ci.yml` file